.github/workflows/backport.yaml

```
@@ -0,0 +1,44 @@
+name: Automatic Backport
+
+on:
+  pull_request_target:
+    types: [closed]   # fires when PR is closed (merged)
```

Why `pull_request_target` and not just `pull_request`? If we need the merge commit, we should probably use the latter

* https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target

> `pull_request_target`: This event runs in the context of the base of the pull request, rather than in the context of the merge commit, as the pull_request event does. 

* https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request

```suggestion
on:
  pull_request_target:
    types: [closed]   # fires when PR is closed (merged)
```